script.js

```diff
@@ -96,7 +100,24 @@ openModal.forEach(function (e) {
 
     inputDate.id = "date";
     inputDate.type = "date";
+    inputDate.valueAsNumber =
+      Date.now() - new Date().getTimezoneOffset() * 60000;
     modal.appendChild(inputDate);
+
+    // Button "search available times"
+    button.innerHTML = "Search available times";
+    button.type = "submit";
```

Comment:
The `submit` type tells the browser that clicking the button should submit the enclosing `<form>`. You don't have a form here, so there is nothing the browser can do with this information. Not sure if there are any accessibility benefits to setting the type to `submit` anyway?